apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -268,43 +315,33 @@ mat-nav-list {
   transform: scale(1);
 }
 
-#mainHeader {
-  height: 70px;
-  overflow: visible;
-}
-
-#mainHeader #mainHeaderLogo A {
-  top: 5px;
-  position: relative;
-  display: block;
-}
-
-svg {
-  overflow: hidden;
-  vertical-align: middle;
+.toolbar {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 65px;
+  background-color: var(--primary-light);
 }
 
-#mainHeader .mainHeaderRight {
-  margin-left: auto;
+#mainHeader {
   display: flex;
-  align-content: flex-end;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: left;
+  margin: 10px;
 }
 
-.mainHeaderNav {
-  display: flex;
-  min-width: 50px;
-  margin-right: 20px;
-  font-size: 1.2rem;
+.paragraph-section {
+  margin: 0;
+  font-size: 1rem;
+  font-weight: 300;
```

Comment:
Maybe you can find a font that fits this case, so we are using the typography system across the board. Look at [this](https://m3.material.io/styles/typography/type-scale-tokens) for inspiration.